File: library/src/main/java/com/vanniktech/rxbilling/LogcatLogger.java
package com.vanniktech.rxbilling;

import android.support.annotation.NonNull;
import android.util.Log;

final class LogcatLogger implements Logger {
  private static final String TAG = "RxBilling";

  @Override public void d(@NonNull final String log) {
    Log.d(TAG, log);
  }

  @Override public void w(@NonNull final String log) {
    Log.w(TAG, log);
  }

  @Override public void w(@NonNull final Throwable throwable) {
    Log.w(TAG, throwable);
  }

  @Override public void e(@NonNull final String log) {
    Log.e(TAG, log);
  }
}

File: rxbilling-google-play-library-v6/src/main/kotlin/com/vanniktech/rxbilling/google/play/library/v6/LogcatLogger.kt
package com.vanniktech.rxbilling.google.play.library.v6

import android.util.Log
import com.vanniktech.rxbilling.Logger

internal class LogcatLogger : Logger {
  override fun d(tag: String, log: String) {
    Log.d(tag, log)
  }

  override fun w(tag: String, throwable: Throwable) {
    Log.w(tag, throwable)
  }
}
